pages/classes/wise.md

```diff
@@ -26,10 +26,165 @@
 
 </header>
 
-Choose an archetype and gain its features. The Wise archetypes include: **[Sorcerer](pages/classes/wise.md?id=sorcerer)**.
+Choose an archetype and gain its features. The Wise archetypes include: [Elementalist](pages/classes/wise.md?id=elementalist) and [Sorcerer](pages/classes/wise.md?id=sorcerer).
 
 <header>
 
+## Elementalist
+
+<p class="subheading">Magical Wise archetype</p>
+
+</header>
+
+You can tap into the raw, primordial forces of nature.
+
+| Level | Archetype Features         | Spell Slots |
+| ----  | -------------------------- | ----------- |
+| 1st   | Minor Elementalism, Rituals, Elementalism, Proficient: Nature, medium armor, shields, and martial weapons | 2  |
+| 2nd   | Primordial Form | 3 |
+
+<header>
+
+<header>
+
+### Minor Elementalism
+
+<p class="subheading">1st-level Elementalist archetype feature</p>
+
+</header>
+
+You can create small, temporary, and harmless sensory effects using the elements, such as hand-sized illusions, whispers, marks or symbols, or small appearance changes.
+
+<header>
+
+<header>
+
+### Rituals
+
+<p class="subheading">1st-level Elementalist archetype feature</p>
+
+</header>
+
+You can cast the rituals below. A ritual takes 10 minutes to cast.
+
+<header>
+
+<section class="spells">
+
+<section class="spell">
+
+<header>
+
+#### Spirit Tongue
+
+<p class="subheading">1st-level Elementalist ritual</p>
+
+</header>
+
+You speak the language of the elements, such as stones, birds or even trees. What they have to say may or may not be helpful. You can ask the spirits for guidance, giving you an additional **d4** on one [Check](../../pages/rules/rolling/checks.md) appropriate for the kind of spirit you are speaking with.
```

Comment:
Too open to spam abuse? Possible fix is to add fictional permission. Otherwise, less powerful than *guidance*.

Reply:
Going to keep as-is, see how it is used in play.